Change JSON BBEs (#3862)
ballerina-platform/ballerina-distribution: examples/casting-json-to-user-defined-type/casting_json_to_user_defined_type.md

# Casting JSON to user-defined type

> In order to access the field in `json` value, the easiest way is to convert `json` value to a user-defined type. 

Reviewer: ```suggestion
In order to access the field in the `json` value, the easiest way is to convert the `json` value to a user-defined type. 
```

> The type-casting can be used to do that. But if cast fails, program panic with an error. However the recommended way to do this is by using langlib functions.

Reviewer: ```suggestion
The type-casting can be used to do that. However, if the cast fails, the program panics with an error. The recommended way to do this is by using langlib functions.
```

> Cast to user defined type will work on mutable structure only if the inherent type(A structured value has an inherent type, which is a type descriptor which is part of the structured value's runtime value.) of that structure is a subtype of user defined type.

Reviewer: ```suggestion
Casting to a user-defined type will work on mutable structure only if the inherent type (a structured value has an inherent type, which is a type descriptor that is part of the structured value's runtime value) of that structure is a subtype of the user-defined type.
```

> Casting immutable values will work but it does not do numeric conversions.

Reviewer: ```suggestion
Casting immutable values will work. However, it does not do numeric conversions.
```
